replicatedhq/ship
# Ship update stable content sha

What I Did
------------
Remove the `.git` directory from the files fetched by go-getter if present. Resolves https://github.com/replicatedhq/ship/issues/485

How I Did it
------------
Pretty much exactly that - I check for the directory, and remove it if present.

How to verify it
------------
Run `ship init https://github.com/replicatedhq/test-charts/tree/master/`, check the content sha in `.ship/state.json`, then run `ship update` and the content sha should have remained the same

Description for the Changelog
------------
Fixed `ship update` on git repo roots


Picture of a Boat (not required but encouraged)
------------


![USS McCampbell (DDG 85)](https://upload.wikimedia.org/wikipedia/commons/d/dc/US_Navy_081007-N-9573A-016_The_guided-missile_destroyer_USS_McCampbell_%28DDG_85%29_is_anchored_at_full_dress_ship_during_the_International_Fleet_Review.jpg "USS McCampbell (DDG 85)")
